app/package.json

```diff
@@ -34,6 +34,8 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-react": "^7.9.4",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
```

Comment:
wait, so it's the app package that dictates merge behaviour for the entire repo? seems weird to me.

Reply:
#JustNodeThings 😉

Reply:
lol we can look at [lerna](https://github.com/lerna/lerna) if we expand to multiple JS projects in this repo

Reply:
thoughts on just shifting commit lint into a repo level package.json, since it's not really appropriate here?